CHANGELOG.rst

```diff
@@ -12,6 +12,10 @@
 * Added documentation related to issue (`#1937`_)
   Thanks `@skylarjhdownes`_ for the PR.
 
+* In ``python.PyobjMixin.getmodpath``, don't assume the module's file extension
```

Comment:
I would write this to reflect the use case instead of the internal implementation detail that changed, perhaps:

> Changed some internal details slightly so it is possible to collect files of any extension as Python modules.